### ISSUE TITLE
Cannot debug a bot with MsAppId and Pass

### ISSUE THREAD

**Issue by srinaath** [microsoft/BotFramework-Composer#8205]
## Describe the bug
Create a bot. Add an msappid and msapppassword. Start the bot. Try running it in Emulator or Integrated Webchat in Composer. It throws a 401 error.

## Version
Current Nightly on Composer. Composer 2.0 works totally fine.

## Browser
Both electron and web

**Comment by tdurnford** [comment]
I ran into a 500 error after I set the app id and password on my Mac - @srinaath is also on a Mac. @EricDahlvang was able to update the app id and password on Windows without any problems so the issue might be operating system dependent.

**Comment by luhan2017** [comment]
I found another issue during the test, when I open the bot the first time, I didn't enter the password, I tried the webchat and emulator both not worked, which is expected. But after I enter the MicrosoftPassword, and restart the bot, the emulator works well, but webchat always responses with 401. There is probabaly an issue that webchat didn't update the appid and password after restarting the bot.
